### DISCUSSION TITLE
Feature Request: add haptic feedback

### DISCUSSION THREAD

**Discussion by ASNNetworks** [kinghat/tabbed-card#37] [Feature Requests]
Adding haptic feedback would be amazing, like custom button-card supports this. Works great on iOS and Android devices. This allows the phone to give a vibration on tab press. Choices are:

success, warning, failure, light, medium, heavy, selection

**Comment by kinghat** [comment]
i will eventually look into this. its not a super high priority atm so i wont give a timeline. thanks for the suggestion!
